sdk/identity/identity/src/credentials/azurePipelinesCredential.ts

```diff
@@ -109,6 +103,7 @@ export class AzurePipelinesCredential implements TokenCredential {
       method: "POST",
       headers: createHttpHeaders({
         "Content-Type": "application/json",
+        "Content-Length": 0,
```

Comment:
Curious - why was this needed / are we sure this is needed?

Reply:
Not needed anymore. I added it when i was debugging the errors when the code wasn't working as expected with the 7.1 version.

Reply:
removed